specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2023-04-01-preview/loadtestservice.json

```diff
@@ -0,0 +1,3301 @@
+          "readOnly": true
+        }
+      }
+    },
+    "ErrorResponseBody": {
+      "description": "The definition of an error object.",
+      "required": [
+        "error"
+      ],
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/Error"
+        }
+      }
+    },
+    "FileInfo": {
+      "description": "File info",
+      "type": "object",
+      "properties": {
+        "fileType": {
+          "description": "File type",
+          "enum": [
+            "JMX_FILE",
+            "USER_PROPERTIES",
+            "ADDITIONAL_ARTIFACTS"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "FileType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "JMX_FILE",
+                "description": "If file is jmx script"
+              },
+              {
+                "value": "USER_PROPERTIES",
+                "description": "If file is user properties"
+              },
+              {
+                "value": "ADDITIONAL_ARTIFACTS",
+                "description": "If file is not any of other supported type"
+              }
+            ]
+          }
+        },
+        "expireDateTime": {
+          "format": "date-time",
+          "description": "Expiry time of the file (ISO 8601 literal format)",
+          "type": "string"
+        },
+        "validationStatus": {
+          "description": "Validation status of the file",
+          "enum": [
+            "NOT_VALIDATED",
+            "VALIDATION_SUCCESS",
+            "VALIDATION_FAILURE",
+            "VALIDATION_INITIATED",
+            "VALIDATION_NOT_REQUIRED"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "FileStatus",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "NOT_VALIDATED",
+                "description": "File is not validated."
+              },
+              {
+                "value": "VALIDATION_SUCCESS",
+                "description": "File is validated."
+              },
+              {
+                "value": "VALIDATION_FAILURE",
+                "description": "File validation is failed."
+              },
+              {
+                "value": "VALIDATION_INITIATED",
+                "description": "File validation is in progress."
+              },
+              {
+                "value": "VALIDATION_NOT_REQUIRED",
+                "description": "Validation is not required."
+              }
+            ]
+          }
+        },
+        "validationFailureDetails": {
+          "description": "Validation failure error details",
+          "type": "string"
+        },
+        "fileName": {
+          "description": "Name of the file.",
+          "type": "string"
+        },
+        "url": {
+          "description": "File URL.",
+          "type": "string"
+        }
```

Comment:
These two properties were at the top of this schema in the last version. Was there some reason to move them?